cli/jobs/deepspeed/deepspeed-training/generate-yml.sh

```diff
@@ -2,22 +2,28 @@
 # Generate key
 ssh-keygen -t rsa -f './src/generated-key' -N ''
 
-# Generate yaml file with key path
+# Pre-set process_count_per_instance so it can be passed into deepspeed via bash script.
```

Comment:
Should we also add that this number is supposed to represent the number of GPUs per VM based on the chosen cluster?

Reply:
I changed the name, so its self-explanatory now